packages/yoastseo/src/languageProcessing/helpers/sentence/SentenceTokenizer.js

```diff
@@ -450,12 +450,16 @@ export default class SentenceTokenizer {
 						"block-end" !== nextToken.type &&
 						"sentence-delimiter" !== nextToken.type &&
 						this.isCharacterASpace( nextToken.src[ 0 ] ) ) {
+						// Don't split on quotation marks unless they're preceded by a full stop.
+						if ( token.src === "\"" && previousToken.src !== "." ) {
```

Comment:
Here (and in one of the lines below, we could use the function `isQuotation` to make sure we target all possible quotation marks.